OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -26,6 +26,8 @@ public class ParatroopersPowerInfo : SupportPowerInfo
 		public readonly int SquadSize = 1;
 		public readonly WVec SquadOffset = new WVec(-1536, 1536, 0);
 
+		public readonly string ReinforcementsArrivedSpeechNotification = null;
```

Comment:
What about adding a `[Desc("Notification to play when entering the drop zone.")]`?